src/locales.json

```diff
@@ -50,14 +50,18 @@
   },
   "categories": {
     "ar": {
```

Comment:
Should we add a placeholder if we see that some translations are missing? e.g. `createPdf`

Reply:
I think this is not necessary since the default is used than which is imo better (Create PDF instead of a possible placeholder createPdf)

Reply:
Maybe we can copy the english text. Yes it falls back but it makes it very difficult to fix missing translations if we don't mark them in some way.

An external list would work too but maybe is difficult to keep up to date.

We can also convert the language files to .js files and include comments.

Reply:
I actually made a script once, that outputs all missing translations to the console... I'll add it to the project :D